### ISSUE TITLE
"Next" button on search results not clickable

### ISSUE THREAD

**Issue by lornajane** [ibm-cds-labs/dW#117]
The search results have a "Next" button at the bottom where there is more than a page's worth of results.  This link isn't "clickable" in the normal sense (I use Vimium on Chrome, see the attached screenshot where Vimium has added labels for everything that can be clicked, and the Next button does not have a label)
![screenshot-20161012201321](https://cloud.githubusercontent.com/assets/172607/19324210/5864ac24-90b8-11e6-99d9-9fab428b92fb.png)

**Comment by vabarbosa** [comment]
addressed
